django-backend/fecfiler/web_services/dot_fec/dot_fec_serializer.py

```diff
@@ -22,11 +22,20 @@ def get_value_from_path(object, path):
     return value
 
 
-def boolean_serializer(model_instance, field_name, mapping):
+def boolean_x_serializer(model_instance, field_name, mapping):
     value = get_value_from_path(model_instance, mapping.get("path", None) or field_name)
     return "X" if value else ""
 
 
+def boolean_yn_serializer(model_instance, field_name, mapping):
```

Comment:
This is great! I knee-jerk reacted to `is True` but this is really well thought out considering the `None` scenario.